plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/OTelTelemetryPlugin.java

```diff
@@ -44,7 +44,8 @@ public List<Setting<?>> getSettings() {
             OTelTelemetrySettings.TRACER_EXPORTER_BATCH_SIZE_SETTING,
             OTelTelemetrySettings.TRACER_EXPORTER_DELAY_SETTING,
             OTelTelemetrySettings.TRACER_EXPORTER_MAX_QUEUE_SIZE_SETTING,
-            OTelTelemetrySettings.OTEL_TRACER_SPAN_EXPORTER_CLASS_SETTING
+            OTelTelemetrySettings.OTEL_TRACER_SPAN_EXPORTER_CLASS_SETTING,
+            OTelTelemetrySettings.TRACER_SPAN_EXPORTER_ENDPOINT
```

Comment:
how is the protocol specified as gRPC?

Reply:
Also OTel allows passing any specific headers, how would those be passed here?

Reply:
The Provider is just a wrapper over existing [OtlpGrpcSpanExporter](https://github.com/open-telemetry/opentelemetry-java/blob/2e2c308293800901aed771c3e56bba71b419257a/exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporter.java#L19). This uses gRPC. 

For passing specific settings, we need to make telemetrysdk as Auto-configurable. We will take that in later PRs https://github.com/opensearch-project/OpenSearch/pull/9453 after opening RFC.

Reply:
We will soon open RFC on how to make it auto configurable. For now, as part of experimental release we just want to give a way to visualise the spans so for now its fixed.

Reply:
Is the change pushed to mark Request tracing as experimental via feature flag?

Reply:
yes

Reply:
Please create a github issue to track this.

Reply:
https://github.com/opensearch-project/OpenSearch/issues/9764